apis/concierge/authentication/v1alpha1/types_tls.go.tmpl

```diff
@@ -1,11 +1,30 @@
-// Copyright 2020-2022 the Pinniped contributors. All Rights Reserved.
+// Copyright 2020-2024 the Pinniped contributors. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 package v1alpha1
 
+// CABundleSource provides a source for CA bundle used for client-side TLS verification.
+type CABundleSource struct {
```

Comment:
Should we maybe call this struct `CertificateAuthorityDataSourceSpec`? What name would be most consistent with existing naming conventions for these types?

Reply:
will be done in next push